SwiftWisdom/Core/StandardLibrary/String/String+Indexing.swift

```diff
@@ -1,7 +1,6 @@
 import Foundation
 
-public extension String {
-    // TODO: Need to write tests for this
```

Comment:
Can you make this TODO more specific instead of removing it? In particular, `ip_range(from range: NSRange) -> Range<String.Index>?` remains untested, as far as i can see. The rest have tests